### ISSUE TITLE
Add Instructions for Optional AaC Command Arguments

### ISSUE THREAD

**Issue by Coffee2Bits** [jondavid-black/AaC#280]
Add instructions for optional arguments that are presented to the user when they execute AaC commands via the VSCode Extension.

## AC:
- [ ] Optional arguments are displayed with additional text indicating that the argument can be skipped via the ESC key.
- [ ] Optional arguments are clearly labaled as optional arguments when presented to the user.

**Comment by cvchaparro** [comment]
The first AC may conflict with what is already displayed by VSCode itself from what I've seen:

![Screen Shot 2022-04-01 at 16 42 07](https://user-images.githubusercontent.com/2924081/161345470-98378e8e-2d7c-4d04-87fe-a0235526c6cc.png)

**Comment by Coffee2Bits** [comment]
> The first AC may conflict with what is already displayed by VSCode itself from what I've seen:
> 
> ![Screen Shot 2022-04-01 at 16 42 07](https://user-images.githubusercontent.com/2924081/161345470-98378e8e-2d7c-4d04-87fe-a0235526c6cc.png)

Good point. I was thinking of including the text "(Escape to skip)" next to the name of the variable. So, for the image it would look like "--output_directory (Optional Argument. Press escape to skip)". Is it redundant? Yes, it is, but I am that terrible user that ignores instructions that are underneath the input field. Since I'm not alone in being such a terrible user, I think it would certainly add clarity for users like me. 🙃

**Comment by lizzcondrey** [comment]
Nothing is changing about code execution for the extension, this is purely a visual display update, so no modifications to the unit tests at this time.

**Comment by lizzcondrey** [comment]
Final confirmation of the "looks" of improvement is blocked by #625.
